plugins/hw_event/gen.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
is this a required file?

Reply:
it is for generating code under pb, but it is not build into the binary.